src/selectors/host/selectors.js

```diff
@@ -1,3 +1,35 @@
 import { get } from 'lodash';
 
-export const getHostStatus = host => get(host, ['metadata', 'labels', 'metalkube.org/operational-status']);
+import { RESPONSE_TO_HOST_STATUS, HOST_STATUS_TO_TEXT } from '../../utils/status/host/constants';
+
+export const getHostStatus = host => {
+  /* Returns a status string based on the available host information.
+   * Possible status strings:
+   *
+   * - Discovered
+   * - Registering
+   * - Registration error
+   * - Validation error
+   * - Inspecting
+   * - Ready
+   * - Making available
+   * - Preparing to provision
+   * - Provisioning
+   * - Provisioned
+   * - Deprovisioning
+   * - Starting maintenance
+   * - Maintenance
+   * - Stopping maintenance
+   * - Deleting
+   */
+
+  const operationalStatus = get(host, 'status.operationalStatus');
+  const provisioningState = get(host, 'status.provisioning.state');
+
+  return RESPONSE_TO_HOST_STATUS[provisioningState || operationalStatus];
```

Comment:
see below, this can be `return provisioningState || operationalStatus;` with below changes

Reply:
Done